src/plugins/embeddable/public/lib/panel/embeddable_panel.test.tsx

```diff
@@ -218,7 +218,7 @@ const renderInEditModeAndOpenContextMenu = async (
 test('HelloWorldContainer in edit mode hides disabledActions', async () => {
   const action = {
     id: 'FOO',
-    type: 'FOO' as ActionType,
+    factoryId: 'FOO' as ActionType,
```

Comment:
Should we also rename this `ActionType` type into something like `StaticActionFactoryId` ?

Reply:
In general, I think we wanted to get rid of this type. But, yes, for staying consistent with the naming would be nice to have it. I would name it `ActionFactoryId`, curious, why `Static*` prefix?

Reply:
Oh, there are no dynamic action factories, right? 😕 
I think I confused it with dynamic actions.. `ActionFactoryId` should be good then

Reply:
`factoryId` is only for dynamic actions, for all "static" actions that fields is actually not used, AFAIK.

Reply:
Done.